engine/install.go

```diff
@@ -40,6 +42,8 @@ type installer struct {
 	keepaliveTimeout time.Duration
 	runner           config.Runner
 	labels           map[string]string
+	dockerUsername         string
+	dockerPassword         string
```

Comment:
these changes can be removed. We have a separate PR that is proposing this change

Reply:
Yea I proposed that PR 😄 . I'll remove these from this PR